src/resty/aws/s3.lua

```diff
@@ -0,0 +1,96 @@
+local AWS = require("resty.aws")
+local normalize = require("kong.tools.uri").normalize
+
+-- this happens at init phase because the kong.clustering.new() is called by kong.init at init phase
```

Comment:
Test suites for this file?

Reply:
This file was mistakenly added. Removing.